From 03261cb6f4b570427ee1d884eb23cc71a3a9475b Mon Sep 17 00:00:00 2001
From: Vinod Kumar <vinod.kumarx.vinod.kumar@intel.com>
Date: Tue, 10 Oct 2017 16:02:02 +0800
Subject: [PATCH 19/96] ASoC: Intel: Skylake: Removed the S24_3LE format
 support from generic driver

Leaf Hill Hardware support the S16_LE, S24_LE and S32_LE format.
Any other needed conversions like S24_3LE are not required to declare in
the kernel driver their conversion are done in user space,

So removed the S24_3LE format support from the driver.

Signed-off-by: Vinod Kumar <vinod.kumarx.vinod.kumar@intel.com>
---
 sound/soc/intel/boards/apli_lhcrb_generic.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/sound/soc/intel/boards/apli_lhcrb_generic.c b/sound/soc/intel/boards/apli_lhcrb_generic.c
index 864b74c..a6a3fb5 100644
--- a/sound/soc/intel/boards/apli_lhcrb_generic.c
+++ b/sound/soc/intel/boards/apli_lhcrb_generic.c
@@ -73,8 +73,7 @@ static int apli_lhcrb_dummy_startup(struct snd_pcm_substream *substream)
 	};
 
 	static u64 formats = SNDRV_PCM_FMTBIT_S16_LE |
-			SNDRV_PCM_FMTBIT_S24_3LE | SNDRV_PCM_FMTBIT_S24_LE |
-						SNDRV_PCM_FMTBIT_S32_LE;
+			SNDRV_PCM_FMTBIT_S24_LE | SNDRV_PCM_FMTBIT_S32_LE;
 
 	static struct snd_pcm_hw_constraint_list hw_constraints_rates = {
 		.count = ARRAY_SIZE(rates),
-- 
1.9.1

